examples/charts/horizon.py

```diff
@@ -25,7 +25,8 @@
 output_file("horizon.html")
 
 hp = Horizon(
-    xyvalues, index='Date',
+    xyvalues, x_names=['Date'],
```

Comment:
`x_names = 'Date'` should also work, and be converted to the list internally. This is the common case I would imagine.